README.md

```diff
@@ -59,21 +59,23 @@ New features should also be reported as [issues]. Please try to describe the fea
 #### Support
 
 * Babbaj
+* Bayleaf
 * BeeVa
 * [Berrely]
 * Ezioblade12
 * FakeAccount1231
 * ForkyFork
+* Impact Bot
 * [living] (GitHub)
 * ObiJuan_
-* sakuya
 * rebane2001
+* Robocop
+* sakuya
+* Scandal Bot
+* Skyklat
```

Comment:
sky has a github with the same name, you can add it the same way the other githubs are linked